plugins/woocommerce/src/Admin/Notes/NoteTraits.php

```diff
@@ -191,4 +199,65 @@ public static function has_note_been_actioned() {
 
 		return false;
 	}
+
+/**
+	 * Update a note field of note1 if it's different from note2 with getter and setter.
+	 *
+	 * @param Note   $note1 Note to update.
+	 * @param Note   $note2 Note to compare against.
+	 * @param string $field_name Field to update.
+	 * @return bool True if the field was updated.
+	 */
+	private static function update_note_field_if_changed( $note1, $note2, $field_name ) {
```

Comment:
It would be nice to have some tests for this function 😆

Reply:
Good idea! Added in b7cee3e867c9c600fedc888b093bfd3e2af6607c.

I think we need to wait for PR https://github.com/woocommerce/woocommerce/pull/32287 to ensure the tests work, though I have tested them in WCA repo.